django/test/testcases.py

```diff
@@ -825,6 +825,10 @@ class TransactionTestCase(SimpleTestCase):
     # are not available, we allow queries to be run.
     allow_database_queries = True
 
+    @classmethod
```

Comment:
I think ideally we could refactor this so that SQLite doesn't get some special treatment (`conn.vendor == 'sqlite'` check) in `TestCase`. Maybe we can leave thing as-is for now (don't introduce this method) since that's outside the scope of this ticket.